### PR TITLE
Add Next.js 13 (beta) guide

### PR DESCRIPTION
**Pull request by bttf**
This adds a guide to our docs on how to get GrowthBook set up w/ Next.js 13's new 'app folder' pattern (which is still in beta). Steps in the guide were followed and tested locally.

image

### CODE REVIEW

File: docs/docs/guide/nextjs-13-and-growthbook.mdx [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+slug: nextjs-13-and-growthbook
+---
+
+# GrowthBook and Next.js 13 (beta)
+
+This document is a guide on how to add GrowthBook feature flags to your Next.js application. It assumes you are starting from scratch, and are using the new 'app directory' pattern introduced in Next 13.
+
+If you already have a NextJS application, you can skip to step 2.
+
+## 1. Create your Next.js App
```
Comment (tinahollygb) [inline docs/docs/guide/nextjs-13-and-growthbook.mdx:14]: Stylistically I wouldn't number the headings:

- it's inconsistent with existing style
- when you add a new section in between other sections, you'll have to update the numbers and all references to any numbers that have changed. I'd link to the sub-section using an anchor instead. Heading ID's should be generated for all headings by Docusaurus.

File: docs/docs/guide/nextjs-13-and-growthbook.mdx [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+description: This guide walks you through using GrowthBook with Next.js 13 (beta).
+sidebar_label: Next.js 13 (beta)
+slug: nextjs-13-and-growthbook
+---
+
+# GrowthBook and Next.js 13 (beta)
+
+This document is a guide on how to add GrowthBook feature flags to your Next.js application. It assumes you are starting from scratch, and are using the new 'app directory' pattern introduced in Next 13.
+
+If you already have a NextJS application, you can skip to step 2.
```
Comment (tinahollygb) [inline docs/docs/guide/nextjs-13-and-growthbook.mdx:12]: Is it NextJS or Next.js? We should probably stick to whatever their branding guidelines are with regards to their naming.

**Review by tinahollygb — COMMENTED** [review]
I don't really think having a whole separate page makes sense. We already have Next.js docs, we could add a subsection in those docs, e.g. **Changes for Next.js v13** or similar. If they already have implemented GrowthBook in their v12 app, it would be helpful to see at a glance how it changes for v13.

Also many of these sections, at a glance, look similar if not identical to the existing React or Next.js docs. This will add more docs for us to maintain when we could cross-link or include a separate section specifically concerning Next.js 13.

File: docs/docs/guide/nextjs-13-and-growthbook.mdx (hunk 2 above)
Comment (bttf) [reply docs/docs/guide/nextjs-13-and-growthbook.mdx:12]: Seems like it's Next.js - updated

**Comment by bttf** [comment]
@tinahollygb Thanks - I went with the separate section approach. Updated

File: docs/docs/guide/nextjs-13-and-growthbook.mdx (hunk 1 above)
Comment (bttf) [reply docs/docs/guide/nextjs-13-and-growthbook.mdx]: Going w/ separate section approach

**Review by tinahollygb — APPROVED** [review]
Awesome, this looks more clear now that `layout.tsx` is highlighted as the main area for changes.

**Pull request merged**